Makefile

```diff
@@ -11,8 +11,9 @@ BINDATA := $(shell find conf | sed 's/ /\\ /g')
 STYLESHEETS := $(wildcard public/less/index.less public/less/_*.less)
 JAVASCRIPTS :=
 
-LDFLAGS += -X "code.gitea.io/gitea/modules/setting.BuildTime=$(DATE)"
-LDFLAGS += -X "code.gitea.io/gitea/modules/setting.BuildGitHash=$(SHA)"
+VERSION = $(shell git describe --tags --always | sed 's/-/+/' | sed 's/^v//')
```

Comment:
Just keep it like it is. The `v` replacement can even be done with plain bash ;)